### DISCUSSION TITLE
Claims should be as much use-case agnostic as possible

### DISCUSSION THREAD

**Discussion by Sgitario** [halkyonio/primaza-poc#231] [Q&A]
At the moment, the Claim entity has the following properties:
- name
- serviceRequested
- type
- database
- category (will be included in https://github.com/halkyonio/primaza-poc/issues/201)
- ... among others

However, the `database` property is too database-specific and should be added using the key-value parameter (part of the credential entity). Also, the `type` property should be populated via a select with a set of options (the ones supported by the Service Binding logic). 

My proposal is:
- Remove the `database` property
- Populate the `type` property using a select in the Claims form.

**Comment by aureamunoz** [comment]
I agree with you that is too database-specific so yes, we should have it in the Credential#params.
Regarding the type, ok for having a dropDown list

**Reply by Sgitario** [reply]
PR: https://github.com/halkyonio/primaza-poc/pull/236
